src/transformers/models/mask2former/modeling_mask2former.py

```diff
@@ -2468,7 +2468,7 @@ def forward(
             transformer_decoder_hidden_states = outputs.transformer_decoder_hidden_states
 
         output_auxiliary_logits = (
-            self.config.use_auxiliary_loss if output_auxiliary_logits is None else output_auxiliary_logits
+            self.config.output_auxiliary_logits if output_auxiliary_logits is None else output_auxiliary_logits
```

Comment:
My original goal is to fix `torchscript` related tests for this model, but it turns out `self.config.output_auxiliary_logits` is the intended value - which fixes the failed tests at the same time.

Reply:
Not 100% sure if this counts as `typo`, so cc @sgugger :-)

Reply:
Thank you, it was changed from `use_auxiliary_logits` to `ouput_auxiliary_logits` and went unnoticed apparently.